### DISCUSSION TITLE
온라인 모각코 이슈 관련 질문

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/Mentoring-Project#83] [General]
![image](https://github.com/fkdl0048/Mentoring-Project/assets/84510455/b9d98b4b-2915-4d3d-b780-d59b50fc7ba4)

모각코에서 코멘트를 달고 해당 이슈를 제 프로젝트에 등록을 해서 일정을 스케줄링하고 있는데 저한테만 보이는줄 알았는데 public으로 변경한 후에는 해당 이슈에 조금 더럽게 보이는 것 같아서 고민입니다..

이러한 로그를 안 뜨게 하는 방법이 있을까요?  

지금 발견해서 죄송합니다..ㅠ

**Comment by jongfeel** [comment]
그냥 보여져도 상관 없을 것 같은데요?
저는 좋아 보입니다.

**Reply by fkdl0048** [reply]
감사합니다..!
